libs/components/modals/src/lib/modules/modal/modal-host.component.ts

```diff
@@ -124,6 +125,12 @@ export class SkyModalHostComponent implements OnDestroy {
       provide: SkyMediaQueryService,
       useExisting: SkyResizeObserverMediaQueryService,
     });
+    params.providers!.push({
+      provide: SkyStackingContextService,
+      useValue: new SkyStackingContextService({
+        zIndex: hostService.getModalZIndex(),
```

Comment:
Modal host is an internal service, so we can refactor to produce an observable value.